src/opnsense/mvc/app/models/OPNsense/Base/Constraints/AllOrNoneConstraint.php

```diff
@@ -53,7 +53,7 @@ public function validate($validator, $attribute): bool
             $countEmpty = 0;
             $countNotEmpty = 0;
             foreach ($Fields as $fieldname) {
-                if (empty((string)$parentNode->$fieldname)) {
+                if ($this->isEmpty($parentNode->$fieldname)) {
```

Comment:
I'm sorry I'm a little confused about this one. What kind of object do we get by `$parentNode->$fieldname`?

Reply:
thanks